submissions/seyifunmiOjo/html-forms/contact.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
```

Comment:
You already created `contact page` for your previous task, why creating it again?